src/app/custom-components/countdown/countdown.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+import * as moment from 'moment';
+
+@Component({
+  selector: 'sw-countdown',
+  templateUrl: './countdown.component.html',
+  styleUrls: ['./countdown.component.scss']
+})
+export class CountDownComponent implements OnInit {
+  @Input() date;
+  finishDate;
+  daysRemaning;
+  hoursRemaning;
+  minutesRemaning;
+  secondsRemaning;
+  isFinsshedDate = false;
```

Comment:
*finished

Reply:
But, maybe you should use: isFinishDate because you have another variable called: finishDate